### PR TITLE
Feature/cluster metrics

### PR DESCRIPTION
**Pull request by maierlars**

### CODE REVIEW

File: 3.7/http/administration-and-monitoring-metrics.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+  * For `arangodb_shards_out_of_sync`
+    * Eventually all shards should be in sync and this value eqal to zero.
+    * It can increase when new collections are created or servers are rotated.
+  * For `arangodb_shards_total_count` and `arangodb_shards_leader_count`
+    * This value should be roughly equal for all servers.
+  * For `arangodb_shards_not_replicated`
+    * This value _should_ be zero at all times. If not, you currently have a single point of failure and data is at risk. Please contact our support team.
+
+_Troubleshoot:_
+The distribution of shards should be roughly eqaul. If not please consider rebalancing shards.
```
Comment (mchacki) [inline 3.7/http/administration-and-monitoring-metrics.md]:
```suggestion
The distribution of shards should be roughly equal. If not please consider rebalancing shards.
```

File: 3.7/http/administration-and-monitoring-metrics.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+Coordinator, DBServer, Agents
+
+_Threshold:_
+  * For `arangodb_scheduler_queue_length`:
+    * Typically this should be 0.
+    * Having an non-zero queue length is not a problem as long as it eventually becomes smaller again. This can happen for example during load spikes.
+    * Having a longer queue results in bigger latencies as the requests need to wait longer before they are executed.
+    * If the queue runs full you will eventually get a `queue full` error.
+  * For `arangodb_scheduler_num_worker_threads` and `arangodb_scheduler_awake_threads`
+    * They should increase as load as increases.
```
Comment (mchacki) [inline 3.7/http/administration-and-monitoring-metrics.md]:
```suggestion
    * They should increase as load increases.
```

File: 3.7/http/administration-and-monitoring-metrics.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+    * If the queue length is non-zero for more than a minute you _should_ see `arangodb_scheduler_awake_threads == arangodb_scheduler_num_worker_threads`. If not, consider contacting our support.
+
+_Troubleshoot:_
+Queuing requests will result in bigger latency. If your queue is growing constantly consider scaling up your system to fit your needs.
```
Comment (mchacki) [inline 3.7/http/administration-and-monitoring-metrics.md]:
```suggestion
Queuing requests will result in bigger latency. If your queue is growing constantly consider scaling up your system to fit your needs. Remember to rebalance shards if you scale up database servers.
```

**Comment by ifcologne** [comment]
- Add Troubleshooting advice to handle `arangodb_shards_not_replicated`. 
- Link documentation if possible

**Comment by Simran-B** [comment]
@maierlars Not sure what Ingo means, can you please talk with him and address his requests?

**Comment by maierlars** [comment]
@Simran-B `arangodb_shards_not_replicated` is described very precisely in the `Threshold` section. I could copy that but that would just duplicate information.

**Comment by Simran-B** [comment]
I guess it can be merged then.

**Pull request merged**